src/Transportation/Transportation.scss

```diff
@@ -88,16 +88,18 @@
 }
 
 .treehacks-transportation-alert {
-  background-color: #686e77;
+  background-color: white;
   width: 100%;
   max-width: 750px;
   margin: 60px;
   padding: 40px 20px;
   color: $text-color;
   text-align: center;
+  border-radius: 5px;
+  border: 2px solid $pink;
```

Comment:
Instead of $pink and $orange, can you make the color names more generic? Such as, $primary-text-color, etc. You can check the variables in the `src/theme` folder for examples, and you can define them there.